sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerClientJavaDocCodeSnippets.java

```diff
@@ -523,4 +524,236 @@ public void beginRecognizeReceiptsWithOptions() throws IOException {
         }
         // END: com.azure.ai.formrecognizer.FormRecognizerClient.beginRecognizeReceipts#InputStream-long-RecognizeReceiptsOptions-Context
     }
+
+    /**
+     * Code snippet for {@link FormRecognizerClient#beginRecognizeBusinessCardsFromUrl(String)}
+     */
+    public void beginRecognizeBusinessCardsFromUrl() {
+        // BEGIN: com.azure.ai.formrecognizer.FormRecognizerClient.beginRecognizeBusinessCardsFromUrl#string
+        String businessCardUrl = "{business_card_url}";
+        formRecognizerClient.beginRecognizeBusinessCardsFromUrl(businessCardUrl)
+            .getFinalResult()
+            .forEach(recognizedBusinessCard -> {
+                Map<String, FormField> recognizedFields = recognizedBusinessCard.getFields();
+                FormField contactNames = recognizedFields.get("ContactNames");
+                if (contactNames != null) {
+                    if (FieldValueType.LIST == contactNames.getValue().getValueType()) {
+                        List<FormField> businessCardItems = contactNames.getValue().asList();
+                        businessCardItems.stream()
+                            .filter(businessCardItem -> FieldValueType.MAP == businessCardItem.getValue()
+                                                                                  .getValueType())
+                            .map(formField -> formField.getValue().asMap())
+                            .forEach(formFieldMap -> formFieldMap.forEach((key, formField) -> {
+                                if ("FirstName".equals(key)) {
+                                    if (FieldValueType.STRING == formField.getValue().getValueType()) {
+                                        String firstName = formField.getValue().asString();
+                                        System.out.printf("First Name: %s, confidence: %.2f%n",
+                                            firstName, contactNames.getConfidence());
```

Comment:
Should this be `firstName.getConfidence()` ?
Same comment for all samples/docsnippets.

Reply:
Yes. It should not be contactNames. It should be formField.getConfidence.